.github/workflows/pr-code-checks.yml

```diff
@@ -137,7 +140,7 @@ jobs:
 
   build:
     runs-on: ubuntu-latest
-    needs: [unittest, lint]
+    needs: [unit-tests, lint]
```

Comment:
we miss `check-coverage` step https://github.com/kyma-project/telemetry-manager/blob/main/.github/workflows/pr-code-checks.yml#L45

Reply:
done